### PR TITLE
Update2 code_rev.js

### PR DESCRIPTION
**Pull request by agoldman12**
At the top of this code is how I am currently solving the problem. It uses three functions, one for each character. It gets a reference to the h2 for the character by id and uses that to create a button. I then toggle the "isHidden" class when the button is clicked.

The bottom of the code uses one function to do the same thing. However, I don't have functionality to toggle the display for one character at a time just yet. This is where I would like to end up.

### CODE REVIEW

**Comment by scheul93** [comment]
Can we walk through this together?

**Comment by agoldman12** [comment]
Absolutely. I have been stuck on that one...getting it all to work with one function

Aaron

File: familyGuy_practice_REV/js/code_rev.js [hunk 1]
```diff
@@ -1,28 +1,64 @@
...
+//show and hide image / paragraph for each character
+function toggleDisplay(){
 
-//example of show and hide
-//https://www.lifewire.com/show-and-hide-text-3467102 (but uses <a> tags and specific id's)
\ No newline at end of file
+    let charInfo = document.getElementsByClassName('charInfo');
+
+    for(var i = 0; i < charInfo.length; i++){
+       charInfo[i].classList.toggle('isHidden');
```
Comment (scheul93) [inline familyGuy_practice_REV/js/code_rev.js]: .toggle('isHidden', charInfo[i].name === showName)

File: familyGuy_practice_REV/js/code_rev.js [hunk 2]
```diff
@@ -1,28 +1,64 @@
...
+      Code below is where I want to go
+      - single function to show/hide all character info
+*************************************************************/
 
+//This code loops through all charinfo divs and toggles isHidden. However it does it to all three at the same time.
+//Need to figure out how to use nextSibling (or something similar).
 
+/*
+//show and hide image / paragraph for each character
+function toggleDisplay(){
```
Comment (scheul93) [inline familyGuy_practice_REV/js/code_rev.js]: toggleDisplay(e)

File: familyGuy_practice_REV/js/code_rev.js [hunk 3]
```diff
@@ -1,28 +1,64 @@
...
+    for(var i = 0; i < charInfo.length; i++){
+       charInfo[i].classList.toggle('isHidden');
+
+    }
+}
+
+//reference for <h2 class="charHead"></h2> that I want to use as a button
+let button = document.getElementsByClassName("charHead");
+for (var i = 0; i < button.length; i++) {
+    button[i].addEventListener('click', toggleDisplay, false);
```
Comment (scheul93) [inline familyGuy_practice_REV/js/code_rev.js]: ('click', function() { toggleDisplay(name, id, ...) }

**Pull request merged**